revoke/revoke.go

```diff
@@ -287,11 +285,16 @@ func sendOCSPRequest(server string, req []byte, issuer *x509.Certificate) (ocspR
 	}
 	resp.Body.Close()
 
-	if bytes.Equal(body, ocspUnauthorised) {
+	switch {
+	case bytes.Equal(body, ocsp.UnauthorizedErrorResponse):
 		return
-	}
-
-	if bytes.Equal(body, ocspMalformed) {
+	case bytes.Equal(body, ocsp.MalformedRequestErrorResponse):
```

Comment:
you can merge multiple case condition together, like case A, B, C, D:, and you can break the line on comma.
If there is a recent update in ocsp package, you may also need to update the vendoring.